src/main/java/liuni/KeyHandler.java

```diff
@@ -66,14 +70,23 @@ private static void setupTwitter(Element element) {
         }
         catch (IOException e) {
             System.out.println("Error occurred when setting up twitter4j.properties.");
-            System.out.println(e.toString());
+            e.printStackTrace();
             System.exit(-2);
         }
         finally {
             try {
                 if (writer != null) {
                     writer.close();
                 }
+                try {
+                    Twitter twitter = TwitterFactory.getSingleton();
+                    twitter.verifyCredentials();
+                }
+                catch (TwitterException e) {
+                    System.out.println("Error occurred; improper credentials for Twitter.");
+                    e.printStackTrace();
+                    System.exit(-1);
```

Comment:
from an application perspective, should you always System.exit when an error occurs?